packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -149,6 +149,11 @@ export class LineSeries extends CartesianSeries<LineContext> {
             props.push(diff(this.processedData));
         }
 
+        const listenerProps: (typeof props)[] =
```

Comment:
Note: There's quite a lot of existing & new code duplication in _lineSeries.ts_ and _scatterSeries.ts_. I've filed [AG-9434](https://ag-grid.atlassian.net/browse/AG-9434) this morning for this.